File: linearSearch/linearSearch.js
function linearSearch(list = [],target){
    for (let i = 0; i < list.length; i++) {
        const e = list[i];
        if(target == e){
            console.log(`item position is ${i}`);
            return i 
            //this is the position of the element in the list
        }
    
    }
    console.log(`value ${target} was not found`);
    return
}

list = [1,2,3,4,5,6,7,8,9,10]
linearSearch(list,9)

File: linearSearch/linearSearch.go
package main

import "fmt"

func linearSearch(list []int, target int) bool {
	for index, value := range list {
		if target == value {
			fmt.Println("the index position of target ", value, "is: ", index)
			return true
		}
	}
	return false
}

func main() {

	list := []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}
	target := 4

	fmt.Println(linearSearch(list, target))

}
